### PR TITLE
Docs/readme

### LINKED ISSUE #63: Update and Enhance CofiBlocks README Documentation
Description:

The current README for CofiBlocks contains detailed information but needs refinement to improve readability, usability, and consistency. The objective is to centralize and structure the documentation, ensuring it effectively communicates the project’s purpose, technical setup, and contribution guidelines.

1. Streamline Project Introduction

- Clearly outline what CofiBlocks is, its mission, and how it operates.
- Ensure concise descriptions of key concepts like Collaborative Business, NFTs, and COFI tokens.

2. Improve Setup Instructions

- Provide clear and error-free steps for project setup and development environment configuration.

3. Centralize Development Resources

- Include detailed instructions for using tools like CofiBlocks Dev Assistant.
- Add information on key technologies like Starknet, Prisma, and Bun.

4. Remove Contribution Guide

- We will create a different file for this.

5. Highlight Roadmap and Team

- Clearly present the project’s roadmap with upcoming milestones.
- Introduce team members with concise bios.

### PR DESCRIPTION
**Pull request by aguilar1x**
- [x] Closes #63 

## Description

Update Documentation in README.md and create other template for contributing guide

## Checklist

- [x] The title of the PR is clear and descriptive.
- [x] All tests have been run and passed.
- [x] Code follows project conventions (Biome formatting, linting).
- [x] Documentation has been updated as needed.
- [x] Commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) style.

## Type of Change

- [ ] Bugfix
- [ ] Feature
- [ ] Refactor
- [ ] Chore
- [x] Documentation
- [ ] Test

### Manual Testing
N/A

### Automated Tests
N/A

## Additional Notes

Update Documentation in README.md and create other template for contributing guide

### CODE REVIEW

**Comment by rvalenciano** [comment]
@aguilar1x this looks great! Can you please add the link to the issue we are solving?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,146 @@
...
+
+- ### Cofi Complementary Currency
+```
+Members of the CofiBlocks pilot community on the Cambiatus app can claim their COFI tokens—a complementary blockchain currency that ensures participation in the Collaborative Business's benefits.
+```
+
+- ### Cofi Distribution
+
+```
+At the end of the first year, all CofiBlocks members will receive a percentage of the pilot's financial results in cryptocurrency based on their accumulated COFI tokens. Members will also have the opportunity to participate in decision-making for future stages of the project.
```
Comment (evgongora) [inline README.md:50]: This is already in the faq section but you can still reference it

File: README.md [hunk 2]
```diff
@@ -1,15 +1,146 @@
...
+Karla contributes her extensive experience in blockchain technology and impact-driven solutions.
+
+### Ranulfo Paiva Sobrinho
+**Co-founder**
+
+Ranulfo's background in blockchain development and collaborative economies helps drive the technical and organizational aspects of CofiBlocks.
+
+#
+
+### 💻 **DEVS**
```
Comment (evgongora) [inline README.md:90]: can you change this to maintainers?

**Review by evgongora — COMMENTED** [review]
Awesome job, just a few changes! :)

File: README.md [hunk 3]
```diff
@@ -46,70 +130,142 @@ Key scripts include:
...
+- fix: A bug fix.
+- docs: Documentation only changes.
+- style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc).
+- refactor: A code change that neither fixes a bug nor adds a feature (improvements of the code structure).
+- perf: A code change that improves the performance.
+- test: Adding missing or correcting existing tests.
+- build: Changes that affect the build system or external dependencies (example scopes: gulp, npm).
+- ci: Changes to CI configuration files and scripts (example scopes: travis, circle).
+- chore: Other changes that don't modify src or test files.
+- revert: Reverts a previous commit.
```
Comment (evgongora) [inline README.md:238]: Since there is already a link to conventional commits can you just do a few examples of types?

**Review by evgongora — COMMENTED** [review]
So far we have just one change, and also Randy will contact you for the docker setup process. Thank you so much! This is almost ready :)

File: CONTRIBUTING.md
Comment (evgongora) [inline CONTRIBUTING.md:1]: Please remove contribution.md since is not part of the issue.

**Review by evgongora — CHANGES_REQUESTED** [review]

**Review by evgongora — APPROVED** [review]

**Comment by evgongora** [comment]
Thank you so much! Great effort :)

**Pull request merged**